### DISCUSSION TITLE
Catalyst Website

### DISCUSSION THREAD

**Discussion by imdevarsh** [CatalystDevOrg/Catalyst#69] [Show and tell]
Made with Tailwind, just like the real catalyst browser.
![image](https://user-images.githubusercontent.com/75148774/147209623-984c3166-3f43-4328-be8c-180daad7b4c8.png)
```html
<div class="p-2">
  <h1 class="text-4xl leading-snug">
    Surf the web with the <br />
    <span class="text-emerald-400 font-bold">browser of the future.</span>
  </h1>
  <br />
  <br />
  <p class="text-xl">This is <b class="text-emerald-400">Catalyst</b>. We're working to make a fully-compatible with all web standards browser for the masses. We always emphasise privacy and security, which is why we are <b>fully open source</b>.</p>
  <br />
  <br />
  <p>Made by JaydenDev and other contributors.</p>
</div>

```
If you want, you all can continue off of it. Or, of course, start from Scratch. Maybe not make a site at all?

**Comment by jdev082** [comment]
https://jaydendev.github.io/freecat is what is had rn

**Reply by jdev082** [reply]
you cant host tailwind on GH-Pages

**Reply by imdevarsh** [reply]
you could use Vercel? and also, you know, we don't have to gitignore it, I wanted the repo to be easy to clone when we have bad wifi

**Reply by imdevarsh** [reply]
there is also cdn.tailwindcss.com
```html
<script src="https://cdn.tailwindcss.com"></script>
```

**Comment by jdev082** [comment]
https://jaydendev.github.io/catalyst
@webdev03

**Reply by imdevarsh** [reply]
Oh cool! I'll work on that repository then.. but I think you should remove me from the organisation, move Catalyst and the website there.

**Reply by jdev082** [reply]
@webdev03 you dont have to work on it if you don't want to

**Reply by imdevarsh** [reply]
I'll work on it later but what about the organisation? Just make me out of it, then you can fully own it, and have the website in it too. Or a mono repo I guess...

**Reply by jdev082** [reply]
@webdev03 real quick, how do I put colored text in a readme

```diff
+ Using this
- For colors
+ Ain't all that great
```

**Reply by imdevarsh** [reply]
I kind of forgot.. maybe use span and HTML.

**Reply by imdevarsh** [reply]
```md
<span style="color: blue;">blue blue in markdown</span>
```

**Comment by imdevarsh** [comment]
I've made a much better design in our organisation.

**Reply by imdevarsh** [reply]
@JaydenDev can I have the ability to install in the org?

**Reply by jdev082** [reply]
@webdev03 someone's close to losing write permissions

**Reply by imdevarsh** [reply]
I mean, in the organisation. I need to get Vercel.

**Reply by hello-smile6** [reply]
Username?

**Reply by imdevarsh** [reply]
github.com/The-Catalyst-Browser-Project

**Reply by hello-smile6** [reply]
@The-Catalyst-Browser-Project
